README.md

```diff
@@ -59,15 +59,85 @@ To create an instance of the service:
    1. Type a unique name for the service instance in the **Service name** field. For example, type `my-service-name`. Leave the default values for the other options.
    1. Click **Create**.
 
+## Visual Recognition
+The process for authenticating with Visual Recognition has changed:
+* For new service instances, authenticate by using IAM. See [Using IAM](#using-iam). Also set the service URL by calling the `set_url()` method of the service instance.
+* For service instances created before May 23, 2018, authenticate by providing the API Key for the service instance. See [Using API Key](#api-key).
+
```

Comment:
We need to tell them to set their URL for IAM authentication to `https://gateway.watsonplatform.net/visual-recognition/api` since the default URL is `https://gateway-a.watsonplatform.net/visual-recognition/api`

For next week's release we need to flip this since the default URL will be `https://gateway.watsonplatform.net/visual-recognition/api` and we need to tell people using the old service to use `https://gateway-a.watsonplatform.net/visual-recognition/api`

Reply:
Got it. Ill make the change

Reply:
@mediumTaj I think after next release we'll remove the text in the READMEs about resetting the URL because the SDKs will set the URL based on the auth method, right? At least, that's what @jsstylos and I were discussing this morning.

@ehdsouza Can you get the URL change propagated to the READMEs in the other SDKs?